$OpenBSD$

Same splitmix init.c fix as in net/simplexmq.
Do not include <sys/random.h> on OpenBSD.

--- a/splitmix-0.1.3.1/cbits-unix/init.c
+++ b/splitmix-0.1.3.1/cbits-unix/init.c
@@ -1,6 +1,8 @@
 #include <stdint.h>
 #include <unistd.h>
+#ifndef __OpenBSD__
 #include <sys/random.h>
+#endif
 
 uint64_t splitmix_init() {
 	uint64_t result;
